ossindex/old_ossindex.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2018 Sonatype Inc.
```

Comment:
So this will die at 1.0 time is the thought?? I'd write up another card or tack into the existing....clean up deprecations before 1.0.

Reply:
https://github.com/sonatype-nexus-community/nancy/issues/126